library/src/tests/resources/state_preparation.qs

```diff
@@ -49,52 +49,52 @@ namespace Test {
         T(qs[1]);
     }
 
-    operation TestPrepareComplex(): Unit {
+    operation TestPrepareComplex() : Unit {
         use q = Qubit[2];
         let c00 = ComplexPolar(0.5, 0.0);
-        let c01 = ComplexPolar(0.5, PI()/4.0);
-        let c10 = ComplexPolar(0.5, PI()/2.0);
-        let c11 = ComplexPolar(0.5, 3.0*PI()/4.0);
+        let c01 = ComplexPolar(0.5, PI() / 4.0);
+        let c10 = ComplexPolar(0.5, PI() / 2.0);
+        let c11 = ComplexPolar(0.5, 3.0 * PI() / 4.0);
         ApproximatelyPreparePureStateCP(0.0, [c00, c01, c10, c11], q);
         DumpMachine();
         Adjoint PrepareComplex(q);
     }
 
-    operation TestPreparationCompletion(): Unit {
+    operation TestPreparationCompletion() : Unit {
         let testCases = [
-            // Test positive coefficients
-            [0.773761, 0.633478],
+        // Test positive coefficients
+        [0.773761, 0.633478],
```

Comment:
Another case where looks like comments on the line following an opening brace have cause a lack of indentation. Lines 65 & 66 should be in one level compared to 64 (interesting that 67 - 69 are indented correctly)

Reply:
Indentation logic will be addressed by a future PR.